dist/rpm/openQA.spec

```diff
@@ -742,4 +761,14 @@ fi
 %{_unitdir}/openqa-continuous-update.*
 %{_datadir}/openqa/script/openqa-continuous-update
 
+%files munin
+%defattr(-,root,root)
+%doc contrib/munin/config/minion.config
+%dir %{_prefix}/lib/munin
+%dir %{_prefix}/lib/munin/plugins
```

Comment:
This is not a usual library. Should it really go under `lib`? Have you checked whether there are already other packages containing Munin plugins and what location those packages use?

Reply:
At least that's the location where the plugins are located that come with munin itself...
There is one plugin I could find that uses `_datadir`: https://build.opensuse.org/package/view_file/security:Zorp:7:0:7/zorp/zorp-7_0-7.spec?expand=1

Reply:
Looks like the munin package in Factory uses this path, indeed. I suppose then it is fine.